docs/sources/developers/plugins/error-handling.md

```diff
@@ -0,0 +1,61 @@
++++
+title = "Error handling"
+type = "docs"
++++
+
+# Error handling
+
+This guide explains how to handle errors in plugins.
+
+## Provide usable defaults
+
+Allow the user to learn your plugin in small steps. By providing a usable default configuration, the user can get started right away, and you can avoid unnecessary error messages early on.
+
+For example, by selecting the first field of a expected type, the panel can display a visualization without any user configuration. If a user explicitly selects a field we use that one, otherwise we default to the first field of type `string` that we find:
```

Comment:
```suggestion
For example, by selecting the first field of an expected type, the panel can display a visualization without any user configuration. If a user explicitly selects a field, then use that one. Otherwise, default to the first field of type `string`:
```

Reply:
I am so confused by your use of "we" here. Is "we" the plugin?